hpx-sys/include/wrapper.h

```diff
@@ -44,4 +34,79 @@ inline std::int32_t disconnect_with_timeout(double shutdown_timeout, double loca
 
 inline std::int32_t finalize() { return hpx::finalize(); }
 
-/*inline std::int32_t stop() { return hpx::stop(); }*/
+inline void hpx_copy(const rust::Vec<int32_t>& src, rust::Vec<int32_t>& dest) {
+    std::vector<int32_t> cpp_src(src.begin(), src.end());
+    std::vector<int32_t> cpp_dest(dest.size());
+
+    hpx::copy(hpx::execution::par, cpp_src.begin(), cpp_src.end(), cpp_dest.begin());
```

Comment:
Please remind me why we cannot use `src.begin()` instead of `cpp_src.begin()` , thanks!

Reply:
we have two vectors `src` and `cpp_src`. cpp_src is the vector that has elements from src ranges that user has defined in rust i.e `src.begin() to src.end()`

Reply:
Please use src instead